homeassistant/components/light/hue.py

```diff
@@ -213,10 +215,13 @@ class HueLight(Light):
 
     def __init__(self, light, request_bridge_update, bridge, is_group=False):
         """Initialize the light."""
+        from rgbxy import Converter, GamutA
+
         self.light = light
         self.async_request_bridge_update = request_bridge_update
         self.bridge = bridge
         self.is_group = is_group
+        self.hue_converter = Converter(GamutA)
```

Comment:
Indeed using the wrong gamut (B instead of A) is what is causing my problems.
I made a few more commits to actually get the gamut type (A, B or C) from the Hue birdge and then use that information to select the right gamut for each bulb.